magento/pwa-studio
# Add to Multiple Wishlist from Cart

## Description

Feature to add an item from the cart to multiple wishlists.

Make sure to test it against a backend that has multiple wishlists enabled.

## Related Issue

Closes PWA-1599

### Verification Stakeholders
@schensley 
@tkacheva 
@dpatil-magento 

### Verification Steps

1. Login.
2. Add an item to the cart.
3. Go to the cart page.
4. Click on Save for Later in the kabab menu and select a wishlist from the list. If nonavailable, create one.
5. Once the mutation is completed, go to favorites list and verify if the item is added.

## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
